cmd/syft/cli/eventloop/tasks.go

```diff
@@ -83,6 +83,8 @@ func generateCatalogFileMetadataTask(app *config.Application) (Task, error) {
 	return task, nil
 }
 
+// SPDX requires that a SHA1 digest be present for all files in the SBOM.
+// This task checks for the config, but also cases where the format is spdx
```

Comment:
Note: I should remove this comment since we're no longer doing this for the user